paper-input-behavior.html

```diff
@@ -384,7 +384,16 @@
         label.id = labelledBy;
       }
       this._ariaLabelledBy = labelledBy;
-    }
+    },
+
+    _onChange: function(event) {
+      // stop the propagation in shady DOM
+      event.stopPropagation();
```

Comment:
Do you remember which behavior it caused?  The `stopPropagation` is required for shady DOM since paper-input will trigger the `change` event for both shadow and shady dom.  I could avoid the `stopPropagation` but that will require a different event name, and I couldn't think of one name as simple as `change`.